src/main.py

```diff
@@ -26,15 +27,27 @@
 logger = logging.getLogger(__name__)
 logger.setLevel(LOGGING_LEVEL)
 
-# We need an authorized token to fetch the API. If you have SSO on your org you will need to grant permission
-# Your app and the Github webhook will share this secret
-# You can create an .env file and place the token in it
-GH_TOKEN = os.environ.get("GH_TOKEN")
+
+# The app can run in one mode or the other
+if os.environ.get("GH_APP_ID"):
+    access_tokens = get_access_tokens()
+    # XXX: We will change this in following PRs
+    TOKEN = access_tokens["armenzg-dev"]
+elif os.environ["GH_TOKEN"]:
+    # We need an authorized token to fetch the API. If you have SSO on your org
+    # you will need to grant permission.
+    # You can create an .env file and place the token in it
+    TOKEN = os.environ["GH_TOKEN"]
+
 # Where to report Github actions transactions
 SENTRY_GITHUB_DSN = os.environ.get("SENTRY_GITHUB_DSN")
 
 app = Flask(__name__)
-handler = EventHandler(token=GH_TOKEN, dsn=SENTRY_GITHUB_DSN)
+handler = EventHandler(
+    secret=os.environ.get("GH_WEBHOOK_SECRET"),
```

Comment:
It seems I forgot to add this in the previous PR, thus, we're currently not checking the secret.